packages/xdl/src/Simulator.js

```diff
@@ -141,6 +141,14 @@ export async function _openSimulatorAsync() {
     Logger.global.info('Opening iOS simulator');
     await spawnAsync('open', ['-a', 'Simulator']);
     await _waitForSimulatorRunningAsync();
+    await _waitForDeviceToBoot();
+  }
```

Comment:
Please run `yarn prettier --write packages/xdl/src/Simulator.js` to fix the formatting of this file. I recommend installing a Prettier plugin in your editor so it formats code automatically.